src/pygeoprocessing/geoprocessing.py

```diff
@@ -2199,16 +2200,17 @@ def warp_raster(
         gdal_warp_options=None, working_dir=None,
         raster_driver_creation_tuple=DEFAULT_GTIFF_CREATION_TUPLE_OPTIONS,
         osr_axis_mapping_strategy=DEFAULT_OSR_AXIS_MAPPING_STRATEGY):
-    f"""Resize/resample raster to desired pixel size, bbox and projection.
+    """Resize/resample raster to desired pixel size, bbox and projection.
 
     Args:
         base_raster_path (string): path to base raster.
         target_pixel_size (list/tuple): a two element sequence indicating
             the x and y pixel size in projected units.
         target_raster_path (string): the location of the resized and
             resampled raster.
-        resample_method (string): the resampling technique, one of
-            ``{_GDAL_WARP_ALGOS_FOR_HUMAN_EYES}``
+        resample_method (string): the resampling technique, one of,
+            'rms | mode | sum | q1 | near | q3 | average | cubicspline |
+            bilinear | max | med | min | cubic | lanczos'
```

Comment:
Could you say more about this change?

My thinking behind making this an f-string was that different GDAL versions have different resample methods, and having a way to detect them programmatically and render them to a string for insertion into the docstring meant that we don't need to maintain it quite as much when these options change.  Having said that, I don't think I verified that the strings would render in the API docs, so maybe this is the only way to have them display?

Reply:
So I changed it because sphinx was not rendering docstrings for these functions at all. It didn't raise any errors/warnings about it, but `pydocstyle` couldn't handle it either: https://github.com/PyCQA/pydocstyle/issues/368

I didn't realize that different GDAL versions have different resample methods, though it makes sense. Unfortunately I don't think the variable in the f-string ever gets expanded, whether you're reading the docs on readthedocs, or in the source code, you never get to see the values. 

Alternatively, we could just say, "must be one of the options supported by `gdal.Warp`", or something like that. What do you think? @phargogh

Reply:
Wow!  That is both neat and highly unexpected.  Sphinx must just be flexing/parsing (but not interpreting), and/or using an old version of the language.  Either way, good to know!

So, for the current docstring, let's just leave it as-is.  I have only ever seen gdal add resample methods (except for the one time they renamed 'nearest' to 'near'), so this is likely to be a subset in future releases.